askama_derive/src/generator.rs

```diff
@@ -15,6 +15,14 @@ use parser::node::{
 use parser::{Expr, Node};
 use quote::quote;
 
+fn insert_variable_at_the_end(args: &mut Vec<Expr<'_>>, var_name: String) {
+    if let Some(Expr::Filter(_, ref mut args)) = args.first_mut() {
```

Comment:
Do we need `ref mut` here? Why don't match ergonomics thingies do their thing?

Reply:
Ah, the old debate: better for reading or writing? I tend to prefer the reading but yes, you're absolutely right, ergonomics handle this case. Updating it.

Reply:
(Personally I don't find the explicit `ref mut` better for reading -- it's noise that I don't need when we're matching against `first_mut()`.)